.github/workflows/release-30_publish_release_draft.yml

```
@@ -41,21 +42,17 @@ jobs:
           sudo apt update
           sudo apt install -y protobuf-compiler
 
-      - name: Build ${{ matrix.binary }} binary
+      - name: Build ${{ matrix.binary[1] }} binary
         run: |
-          if [[ ${{ matrix.binary }} =~ chain-spec-builder ]]; then
-            cargo build --locked --profile=production -p staging-${{ matrix.binary }} --bin ${{ matrix.binary }}
-            target/production/${{ matrix.binary }} -h
-          else
-            cargo build --locked --profile=production -p ${{ matrix.binary }}
-            target/production/${{ matrix.binary }} --version
-          fi
+          cargo build --locked --profile=production -p ${{ matrix.binary[0] }} --bin ${{ matrix.binary[1] }}
+          target/production/${{ matrix.binary }} --version
+          target/production/${{ matrix.binary }} -h
```

I think we need to update these lines to actually check the binary.

```suggestion
          target/production/${{ matrix.binary[1] }} --version
          target/production/${{ matrix.binary[1] }} -h
```

---

@pepoviola ah, right, my bad, thank you :)